sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -176,55 +176,55 @@ def process(self, element, lower_bound, upper_bound=float('inf')):
     # [END model_pardo_side_input_dofn]
     self.assertEqual({'a', 'bb', 'ccc'}, set(small_words))
 
-  def test_pardo_with_side_outputs(self):
-    # [START model_pardo_emitting_values_on_side_outputs]
+  def test_pardo_with_tagged_outputs(self):
+    # [START model_pardo_emitting_values_on_tagged_outputs]
```

Comment:
web site needs to be updated for these changes in snippet names. Please sync with: @hadarhod87 @melap